Demo/AppDelegate.swift

```diff
@@ -1,16 +1,20 @@
 import UIKit
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate {
+class AppDelegate: UIResponder {
     var window: UIWindow?
+}
+
+extension AppDelegate: UIApplicationDelegate {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        window = UIWindow(frame: UIScreen.main.bounds)
+        guard let window = self.window else { fatalError("Window not found") }
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-        let navigationController = UINavigationController(rootViewController: ViewController())
-        let window = UIWindow()
```

Comment:
Because of this the screen was shown black and it was freaking me out. Looks like you have to provide a frame for initialization, maybe you can skip it by adding some constrains to the window.

```swift
window = UIWindow(frame: UIScreen.main.bounds)
```